app/views/candidate_interface/referees/confirm_cancel.html.erb

```diff
@@ -1,5 +1,4 @@
 <% content_for :title, t('page_titles.referee_cancel') %>
-<% content_for :before_content, govuk_back_link_to(candidate_interface_review_referees_path, 'Back') %>
```

Comment:
I think this should still be in there right?

Reply:
Oh right, perhaps deleted by accident. Can't do any harm though.